types/stream-chat/api-response-tests/response-generators/channel.js

```diff
@@ -75,6 +75,16 @@ async function deleteChannel() {
 	return await channel.delete();
 }
 
+async function truncateChannel() {
```

Comment:
but generator is added ?

Reply:
Yeah thats fine. Once backend fix is ready, we will use it anyways!